standalone/src/models/shipping-method/zone-rate/zone-rate-draft/fields-config.ts

```diff
@@ -0,0 +1,21 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { KeyReferenceDraft } from '@/models/commons';
+import { ShippingRateDraftRest } from '../../shipping-rate/shipping-rate-draft';
+import type { TZoneRateDraftRest, TZoneRateDraftGraphql } from '../types';
+
```

Comment:
It would help to leave a comment here with the link to this entity's docs:
```
// https://docs.commercetools.com/api/projects/shippingMethods#zoneratedraft
```